Sources/Shared/Models/VIMPictureCollection.m

```diff
@@ -27,6 +27,12 @@
 #import "VIMPictureCollection.h"
 #import "VIMPicture.h"
 
+@interface VIMPictureCollection ()
+
+@property (retain, nonatomic) NSNumber *active;
```

Comment:
Is there a reason why we're using `retain` here instead of `strong`?

Reply:
Ah, not really, just reflex. Learned Objective-C in the pre-ARC days when `retain` was the only option. I think `strong` is the same, but I'll switch this for consistency.
```suggestion
@property (strong, nonatomic) NSNumber *active;
```